File: twitchVotingOverlay/src/chaosWSClient/iVoteOption.ts
export interface IVoteOption {
	/**
	 * Name for the option
	 */
	label: string;
	/**
	 * How meany votes the option has
	 */
	votes: number;
	/**
	 * What messages should result in voting
	 * for this option.
	 *
	 * An example for this would be: ["1", "a"]
	 */
	matches: string[];
}

File: ChaosPipe/IVoteOption.cs
﻿using System.Collections.Generic;

namespace ChaosPipe
{
    public interface IVoteOption
    {
        /// <summary>
        /// Label (name) of the voting option
        /// </summary>
        string Label { get; set; }
        /// <summary>
        /// What strings should be counted as voting for this option (e.g. 1, 5, etc)
        /// </summary>
        List<string> Matches { get; }
        /// <summary>
        /// How many votes this voting option has
        /// </summary>
        int Votes { get; set; }
    }
}
